sky/skylet/providers/gcp/node.py

```diff
@@ -290,6 +291,16 @@ def create_instance(
         """
         return
 
+    @abc.abstractmethod
+    def resize_disk(
+        self, base_config: dict, instance_name: str, wait_for_operation: bool = True
+    ) -> Tuple[dict, str]:
+        """Resize a Google Cloud disk based on the provided configuration.
+
+        Returns the response of resize operation.
+        """
+        return
```

Comment:
raise NotImplementedError?

Reply:
None of the abstract methods have been implemented with a `NotImplementedError`. Should we add this exception to all of them?

Reply:
Good point! Aligning with the current implementation is a better idea. Lets keep the `return` here 🫡